conf.json.template

```diff
@@ -13,5 +13,13 @@
 	"http.port":8080,
 	"http.host":"localhost",
 	"http.endpoint":"/api/v3",
-	"http.port.management":13371
+	"http.port.management":13371,
+	"salt": "cyface_salt",
+	"upload.expiration": 600,
+	"measurement.payload.limit": 100,
```

Comment:
Are you sure this value is correct?
In the old code the default value for 100 MB was:

```
/**
     * The default number of seconds the JWT authentication token is valid after login.
     */
    private static final int DEFAULT_PAYLOAD_LIMIT = Math.toIntExact(BYTES_IN_ONE_MEGABYTE * 100L);
```

With `BYTES_IN_ONE_MEGABYTE = 1_048_576L`

Reply:
This is just some example value and needs to be changed. The problem is that I did not find documentation on whether this is supposed to be in Bytes, Kilotbytes or Megabytes. But if it is Bytes it should probably be changed.